utils/gyb_syntax_support/StmtNodes.py

```diff
@@ -273,6 +274,13 @@
              Child('Colon', kind='ColonToken'),
          ]),
 
+    # switch-unknown-label -> 'unknown' ':'
+    Node('SwitchUnknownLabel', kind='Syntax',
+         children=[
+             Child('UnknownKeyword', kind='Token'),
+             Child('Colon', kind='ColonToken'),
+         ]),
```

Comment:
This is not used. Can I remove this?

Reply:
Whoops, I thought I took it out already. Yes, sorry!